File: CSW2/Assignment_3/Q6.java
public class Q6 {
    public static void main(String[] args) {
        try {
            int n = Integer.parseInt(null);
        } catch (NumberFormatException e) {
            System.out.println(e);
        }
        System.out.println("End of program");
    }

}

/*
 * Output:
 * java.lang.NumberFormatException: Cannot parse null string
 * End of program
 */

File: PPWC/Minor_Assignment_6/Q6.c
#include <stdio.h>
int f(int n){
    static int i = 1 ;
    if (n >=5) return n;
    n = n+1;
    i++;
    return f(n);
}
int main(){
    printf("%d\n", f(1));
    return 0;
}

/*
Output--
5
*/